neo/Ledger/Blockchain.cs

```diff
@@ -405,8 +405,15 @@ protected override void OnReceive(object message)
                 case Block block:
                     Sender.Tell(OnNewBlock(block));
                     break;
+                case Transaction[] transactions:
+                    {
+                        // This message comes from a mempool's revalidation, already relayed
+                        foreach (var tx in transactions)
+                            Sender.Tell(OnNewTransaction(tx, false));
```

Comment:
We don't need to call `Sender.Tell()` here. Because the sender is `ActorRefs.NoSender`.